### ISSUE TITLE
Template for documentation issue

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/expressjs.com#1560]
Could it be a good idea to add a template for documentation similar to what biomejs does ([ref](https://github.com/biomejs/website/blob/main/.github/ISSUE_TEMPLATE/01_documentation.yml)), either just for the expressjs.com repository or for the entire organization?

If you think it's a good idea, I could create a PR to add it.

**Comment by crandmck** [comment]
Yes, a doc PR template is a good idea.  Also add a note that this is specifically for DOCUMENTATION issues, not issues or bugs with the software itself.  Often, this is an area of confusion.

**Comment by chrisdel101** [comment]
Is this possible without admin access to the site? I looked into it and found https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository. You prob found this long ago alredy.
It looks like non-admins can only add forms, and not the nice input field templates that we'd more likely want. I don't know we'd want a form, do we?

We (as non-admins) might be able to add things by cloning this https://github.com/expressjs/.github. Maybe this would at least allow the creating of a documentation item in the list https://github.com/expressjs/.github/tree/master/.github/ISSUE_TEMPLATE? Not sure if this would work but I want to look into it.

**Comment by blakeembrey** [comment]
@bjohansebas Would it make more sense for this to be in the repo instead of globally? The contents of https://github.com/expressjs/.github/pull/5/files seem to imply that it's more website specific and doesn't translate great to each module (which may not even have documentation pages).

**Comment by bjohansebas** [comment]
I think it's better for it to be global rather than just for the website, since the documentation for the packages is in the READMEs of each package. Any suggestion that can be applied to all packages in general is welcome.
